### DISCUSSION TITLE
Export .sql on save?

### DISCUSSION THREAD

**Discussion by dekiesel** [pgmodeler/pgmodeler#1846] [Q&A] [answered]
Hi,

Is there a way to automatically create an export on save? My use case is that I'd like to test the model after pushing it to my git repo by standing up a postgres db in my git pipeline and then running some tests against that DB.

I could save the export manually, but that means there is a chance that I'll forget that before pushing.

**Comment by rkhaotix** [comment, marked as answer]
Hi @dekiesel 

You can use ```pgmodeler-cli``` and call it from a git hook to trigger the export once you push the model. You can even use the ```--simulate``` option of ```pgmodeler-cli``` to validate the SQL of the model. 

https://pgmodeler.io/support/docs/613-command-line-interface?v=1.1.0
